controllers/ArticleController.js

```diff
@@ -99,7 +99,7 @@ export default class ArticleController {
           { // Article Comments
             model: Comment,
             as: 'articlecomment',
-            attributes: ['body', 'likes', 'createdAt', 'updatedAt'],
+            attributes: ['body', 'likes', 'highlightedText', 'containsHighlightedText', 'createdAt', 'updatedAt'],
             include: [
```

Comment:
So sorry guys. I see you've implemented it here in the getArticleBySlug method. My bad